js/version.ts

```diff
@@ -0,0 +1,15 @@
+// Copyright 2018-2019 the Deno authors. All rights reserved. MIT license.
+// tslint:disable-next-line:no-reference
+/// <reference path="./const.d.ts" />
```

Comment:
Instead of this, I would just add it to the globals in `globals.ts`:

https://github.com/denoland/deno/blob/4dc4329e270f617697154ca62a828cce3b46b348/js/globals.ts#L35-L38

Then you won't need an additional type declaration file.

Reply:
The situation seems a little tricky here. rollup-plugin-replace replaces the occurence even in type declaration. So `const TS_VERSION: string` becomes `const "3.2.1": string;` before passed to typescript.

I found that rollup-plugin-replace doesn't do any parsing of the source code. So we can do something like: `{ version: "TS_VERSION" }`. This seems better because it doesn't require any type declaration. So I swtich to this pattern.